experiment/measurer/measure_manager.py

```diff
@@ -396,6 +397,82 @@ def generate_profdata(self, cycle: int):
             self.logger.error(
                 'Coverage profdata generation failed for cycle: %d.', cycle)
 
+    def update_coverage_state(self, cycle):
+        """Update the previous trial-specific coverage data with coverage
+        information from this cycle."""
+
+        # TODO Recover time stamp from cycle (how long is a cycle?)
+        time_stamp = cycle
```

Comment:
We can just pass the `this_time` (time elapsed since the start of experiment) as a parameter and then we would not need to work with cycles and time, just time would be enough.

Reply:
Done.